### Paramteres used by the controller node ###
controller_node:
  ros__parameters:
    ######################################################
    # AUV
    mass: 35.5
    volume: 0.0364
    r_cog: [0.0, 0.0, 0.0]
    r_cob: [0.0, 0.0, -0.12489]
    Ma: [13.0, 20.0, 35.0, 12.6, 12.7, 7.3]
    Dlinear:  [-22.8, -30.95, -50.26, -16.05, -16.73, -5.13]
    Dquad: [ -28.43, -55.98, -137.5, -0.0, -0.0, -0.00]
    Ib: [0.8061, 0.8, 1.5599, -0.0059, 0.0005, -0.0113]
    #######################################################
    # Controller
    Q: [5.0, 5.0, 5.0, 5.0, 5.0, 5.0, 1.0, 1.0,1.0, 1.0, 1.0, 1.0]
    R: [0.01, 0.01, 0.01, 0.01, 0.01, 0.01]
    error_max:  [0.8, 0.8, 0.8, 0.5, 0.5, 0.5, 0.3, 0.3, 0.3, 0.3, 0.3, 0.3]
    tau_max: [45.0, 45.0, 45.0, 35.0, 35.0, 35.0]
    ######################################################
    # Trajectory generation
    max_translation_speed: 0.7
    max_rotation_speed: 0.45
    max_translation_acc: 0.45
    max_rotation_acc: 0.33
    max_translation_jerk: 0.001
    max_rotation_jerk: 0.002
    #####################################################
    # Line of sight steering
    delta: 2.0
    dt: 0.1
    kappa: 0.1
    radius_of_acceptance: 0.2
    T: 0.1



    
    
    

    
